bridge_adaptivity/api/models.py

```diff
@@ -11,28 +12,39 @@ class OAuthClient(models.Model):
     Content source API connections.
     """
 
-    AUTH_CODE = "authorization code"
-    IMPLICIT = "implicit"
-    PASSWORD = "resource owner password-based"
-    CREDENTIALS = "client credentials"
+    # Note (braiko) commented grant types are types that can exist, but now not used.
+    AUTH_CODE = "code"
+    # IMPLICIT = "implicit"
```

Comment:
this is not a good practice to pull not used code, please add unused parameters to the comment lines.